gen/atlas.tmpl

```diff
@@ -33,5 +36,8 @@ jobs:
       - uses: ariga/atlas-sync-action@v0
         with:
           dir: '{{ .Path }}'
+{{- with .DirName }}
+          dir-name: '{{ . }}'
```

Comment:
```suggestion
          name: '{{ . }}'
```
In sync action, this is named as 'name'.

Reply:
https://github.com/ariga/atlas-sync-action/blob/master/action.yml#L10

Reply:
Oh didn't notice that!, Thanks @a8m .